docs/configuration.rst

```diff
@@ -664,3 +664,35 @@ Configuring the KSQL CLI
 * Additional CLI options can be appended to the command as necessary. See :ref:`the KSQL CLI documentation <install_ksql-cli>` for the KSQL CLI options you can use.
 * The KSQL CLI cannot be configured via environment variables, unlike the KSQL Server image.
 
+-------------------------------
+Confluent Kafka MQTT
+-------------------------------
+
+For the Kafka MQTT image, use variables prefixed with ``KAFKA_MQTT_`` with an underscore (``_``) separating each word instead of periods. As an example, to set ``bootstrap.servers``, ``topic.regex.list`` you'd run the following:
+
+  .. sourcecode:: bash
+
+    docker run -d \
+      --name=cp-kafka-mqtt \
+      --net=host \
```

Comment:
examples should not use hot networking, since it does not work with docker for mac or windows

Reply:
@xvrl as you may confirm, this is not specific to this docker image. All of our example are using the above setting. Thus, a fix should be applied to all of them and should be the subject of a separate PR. The choice of @sansanichfb not to diverge here seems correct to me.

Reply:
Sure, I remember this fix in quick-starts. I also understand that this file was missed on that PR. We'll need a follow-up PR to fix it here too. Piggy-backing it in this PR can be misleading IMO. Good thing is, it's just docs so there's no code freeze for this change.